test/RequestHandlerSwooleRunnerFactoryTest.php

```diff
@@ -53,6 +54,14 @@ public function setUp()
         $this->container
             ->get('config')
             ->willReturn([]);
+
+        $this->container
+            ->has(LoggerInterface::class)
+            ->willReturn(false);
+
+        $this->container
+            ->get(LoggerInterface::class)
+            ->shouldNotBeCalled();
     }
```

Comment:
This test exercises the new `StdoutLogger`, and making the above changes ensures that existing tests run as they did previously.